tools/remove_project_hold.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+// Allow authorized users to remove a hold on a project.
+
+$relPath="./../pinc/";
+include_once($relPath.'base.inc');
+include_once($relPath.'Project.inc'); // validate_projectID() Project() remove_holds()
+include_once($relPath.'metarefresh.inc'); // metarefresh()
+
+require_login();
+
+$projectid = validate_projectID('projectid', @$_POST['projectid']);
+
+$project = new Project($projectid);
+if (!$project->can_be_managed_by_current_user)
+{
+    echo "<p>", _('You are not authorized to manage this project.'), "</p>\n";
```

Comment:
I think in most other places in the code we just use die() instead:
```
die(_('You are not authorized to manage this project.'));
```
We don't actually open an HTML page so I don't think we need to bother with the paragraph tags.